AsyncDisplayKit/ASMapNode.mm

```diff
@@ -232,6 +252,13 @@ - (void)takeSnapshot
   }];
 }
 
+- (UIImage *)defaultPinImageWithCenterOffset:(CGPoint *)centerOffset
+{
+  MKAnnotationView *pin = [[MKPinAnnotationView alloc] initWithAnnotation:nil reuseIdentifier:@""];
+  *centerOffset = pin.centerOffset;
```

Comment:
Instead of instantiating an MKPinAnnotationView every time (which can get very expensive!) let's make this a class method `+ (UIImage *)defaultPinImageWithCenterOffset:` and wrap the body of it in a dispatch_once (see my last comment for example code).